docs/front-end/communities/nl-design-system/getting-started.md

```diff
@@ -0,0 +1,156 @@
+---
+tags:
+  - "open-source"
+  - "front-end"
+  - "accessibility"
+  - "nl-design-system"
+title: "Getting Started: NL Design System"
+---
+
+# Getting Started
+
+Wil je snel aan de slag met NL Design System? In deze handleiding laten we je zien hoe je het lokaal kunt opzetten en hoe je experimenteert met de beschikbare componenten.
+
+## 📌 Benodigdheden
+
+- [Node @ v22.13.1](https://nodejs.org/en/download)
```

Comment:
Hoe belangrijk is deze specifieke versie? Misschien ook verwijzen naar een artikel over `nvm` om het gemakkelijker te maken?

Reply:
Lijkt mij wel slim om deze te pinnen. Anders krijg je de situatie dat Vite niet werkt oid. Maakt de tutorial wat bestendiger voor de lange termijn